patch/0011-arm64-dts-marvell-Add-Nokia-7215-IXS-A1-board.patch

```diff
@@ -11,17 +11,17 @@ Signed-off-by: Natarajan Subbiramani <natarajan.subbiramani.ext@nokia.com>
 Tested-by:  Natarajan Subbiramani <natarajan.subbiramani.ext@nokia.com>
```

Comment:
As it’s not simply copied anymore, please update the commit message description regarding the removal of the eMMC mode HS400.